docs/source/torch.compiler.rst

```diff
@@ -64,9 +66,9 @@ Some of the most commonly used backends include:
    * - Backend
      - Description
    * - ``torch.compile(m, backend="onnxrt")``
-     - Uses ONNXRT for inference on CPU/GPU. `Read more <https://onnxruntime.ai/>`__
+     - Uses ONNX Runtime for inference on CPU/GPU. :doc:`Read more <onnx_dynamo_onnxruntime_backend>`
    * - ``torch.compile(m, backend="tensorrt")``
-     - Uses ONNXRT to run TensorRT for inference optimizations. `Read more <https://github.com/onnx/onnx-tensorrt>`__
+     - Uses ONNX Runtime to run TensorRT for inference optimizations. `Read more <https://github.com/onnx/onnx-tensorrt>`__
```

Comment:
I think this is unrelated to ORT.

Reply:
Yeah, I found this to be tricky
My understanding is that we don't want to have references to `ONNXRT`, thus renamed to ONNX Runtime even for a backend that is not onnx runtime. Maybe the `onnx-tensorrt` backend uses ort as part of their implementation and that is why they listed ONNXRT in their doc? In that case, it would be ok to rename as it is done.

FYI @wschin @abock

Reply:
I am with Thiago. This change doesn't affect the original semantic, so we can keep it.